--- ocserv-1.0.1/src/Makefile.am.orig	2020-05-12 20:57:37.308459654 +0200
+++ ocserv-1.0.1/src/Makefile.am	2020-05-12 21:05:33.645879112 +0200
@@ -70,13 +70,13 @@
 HTTP_PARSER_SOURCES = http-parser/http_parser.c http-parser/http_parser.h
 endif
 
-ocserv_LDADD = ../gl/libgnu.a libccan.a libcommon.a
+ocserv_LDADD = ../gl/libgnu.a libccan.a libcommon.a libipc.a
 ocserv_LDADD += $(LIBGNUTLS_LIBS) $(PAM_LIBS) $(LIBUTIL) \
 	$(LIBSECCOMP) $(LIBWRAP) $(LIBCRYPT) $(NEEDED_HTTP_PARSER_LIBS) \
 	$(NEEDED_LIBPROTOBUF_LIBS) $(LIBSYSTEMD) $(LIBTALLOC_LIBS) \
 	$(RADCLI_LIBS) $(LIBLZ4_LIBS) $(LIBKRB5_LIBS) \
 	$(LIBTASN1_LIBS) $(LIBOATH_LIBS) $(LIBNETTLE_LIBS) \
-	$(LIBEV_LIBS) libipc.a $(NEEDED_LIBPROTOBUF_LIBS) \
+	$(LIBEV_LIBS) $(NEEDED_LIBPROTOBUF_LIBS) \
 	$(CODE_COVERAGE_LDFLAGS) 
 	
 if ENABLE_OIDC_AUTH
@@ -122,9 +122,9 @@
 occtl_occtl_SOURCES += occtl/geoip.c
 endif
 
-occtl_occtl_LDADD = ../gl/libgnu.a libcommon.a $(LIBREADLINE_LIBS) \
-	$(LIBNL3_LIBS) $(NEEDED_LIBPROTOBUF_LIBS) $(LIBTALLOC_LIBS) libccan.a \
-	libipc.a $(NEEDED_LIBPROTOBUF_LIBS) $(CODE_COVERAGE_LDFLAGS) \
+occtl_occtl_LDADD = ../gl/libgnu.a libcommon.a libccan.a libipc.a $(LIBREADLINE_LIBS) \
+	$(LIBNL3_LIBS) $(NEEDED_LIBPROTOBUF_LIBS) $(LIBTALLOC_LIBS) \
+	$(NEEDED_LIBPROTOBUF_LIBS) $(CODE_COVERAGE_LDFLAGS) \
 	$(LIBNETTLE_LIBS) $(MAXMIND_LIBS) $(GEOIP_LIBS)
 
 # Ocpasswd
