--- ocserv-1.1.2/src/Makefile.am.orig	2020-08-11 23:13:48.000000000 +0200
+++ ocserv-1.1.2/src/Makefile.am	2020-12-12 12:01:01.742091938 +0100
@@ -80,13 +80,13 @@
 HTTP_PARSER_SOURCES = http-parser/http_parser.c http-parser/http_parser.h
 endif
 
-CORE_LDADD = ../gl/libgnu.a libccan.a libcommon.a
+CORE_LDADD = ../gl/libgnu.a libccan.a libcommon.a libipc.a
 CORE_LDADD += $(LIBGNUTLS_LIBS) $(PAM_LIBS) $(LIBUTIL) \
 	$(LIBSECCOMP) $(LIBWRAP) $(LIBCRYPT) $(NEEDED_HTTP_PARSER_LIBS) \
 	$(NEEDED_LIBPROTOBUF_LIBS) $(LIBSYSTEMD) $(LIBTALLOC_LIBS) \
 	$(RADCLI_LIBS) $(LIBLZ4_LIBS) $(LIBKRB5_LIBS) \
 	$(LIBTASN1_LIBS) $(LIBOATH_LIBS) $(LIBNETTLE_LIBS) \
-	$(LIBEV_LIBS) libipc.a $(NEEDED_LIBPROTOBUF_LIBS) \
+	$(LIBEV_LIBS) $(NEEDED_LIBPROTOBUF_LIBS) \
 	$(CODE_COVERAGE_LDFLAGS) 
 	
 if ENABLE_LATENCY_SUPPORT
@@ -134,9 +134,9 @@
 occtl_occtl_SOURCES += occtl/geoip.c
 endif
 
-occtl_occtl_LDADD = ../gl/libgnu.a libcommon.a $(LIBREADLINE_LIBS) \
-	$(LIBNL3_LIBS) $(NEEDED_LIBPROTOBUF_LIBS) $(LIBTALLOC_LIBS) libccan.a \
-	libipc.a $(NEEDED_LIBPROTOBUF_LIBS) $(CODE_COVERAGE_LDFLAGS) \
+occtl_occtl_LDADD = ../gl/libgnu.a libcommon.a libccan.a libipc.a $(LIBREADLINE_LIBS) \
+	$(LIBNL3_LIBS) $(NEEDED_LIBPROTOBUF_LIBS) $(LIBTALLOC_LIBS) \
+	$(NEEDED_LIBPROTOBUF_LIBS) $(CODE_COVERAGE_LDFLAGS) \
 	$(LIBNETTLE_LIBS) $(MAXMIND_LIBS) $(GEOIP_LIBS)
 
 # Ocpasswd
